frontend/server/src/UITools.php

```diff
@@ -7,6 +7,20 @@ class UITools {
     public static $isLoggedIn = false;
     /** @var bool */
     public static $isAdmin = false;
+    /** @var bool */
+    public static $isQualityReviewer = false;
+    /** @var string */
+    public static $currentUsername = '';
+    /** @var string */
+    public static $gravatarURL51 = '';
+    /** @var array */
```

Comment:
```suggestion
    /** @var string[] */
```

puedes ser más específico con el tipo.

Reply:
Listo